backend/src/test/java/com/javahelp/backend/data/DynamoDBSurveyResponseStoreTest.java

```diff
@@ -0,0 +1,207 @@
+package com.javahelp.backend.data;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assume.assumeTrue;
+
+import com.amazonaws.regions.Regions;
+import com.javahelp.model.survey.Survey;
+import com.javahelp.model.survey.SurveyQuestion;
+import com.javahelp.model.survey.SurveyQuestionResponse;
+import com.javahelp.model.survey.SurveyResponse;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.User;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DynamoDBSurveyResponseStoreTest {
+    String tableName = "javahelpBackendFilledSurveys";
+
+    Regions regions = Regions.US_EAST_1;
+
+    DynamoDBSurveyResponseStore db = new DynamoDBSurveyResponseStore(tableName, regions);
+
+    @Test(timeout = 5000)
```

Comment:
Is the timeout suppose to be this long in this case?

Reply:
For Dynamo, I think 5s is good yeah.